container/worker/run_openqa_worker.sh

```diff
@@ -6,4 +6,16 @@ if [[ -z $OPENQA_WORKER_INSTANCE ]]; then
 fi
 
 /root/qemu/kvm-mknod.sh
+
+if [ -e "/dev/kvm" ]; then
+  if [ ! $(getent group kvm) ]; then
+    groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  fi
+
+  usermod -a -G kvm _openqa-worker
```

Comment:
This line should also only be executed if the group `kvm` exists (see https://github.com/os-autoinst/openQA/pull/3715#discussion_r575214849).

Reply:
done